test/test_helper.exs

```diff
@@ -137,6 +137,37 @@ defmodule Phoenix.LiveDashboardTest.Endpoint do
   plug Phoenix.LiveDashboardTest.Router
 end
 
+defmodule Phoenix.LiveDashboard.SystemInfoTest.ProcessFilter do
```

Comment:
When a module is only used in a file test, I like to move it to the test file itself so you don't need to jump to another file to understand the test entirely.

Reply:
Now this module is being used in 2 places, so I guess we could keep it where it is :-)